app/javascript/app/utils/uswds.js

```diff
@@ -0,0 +1,31 @@
+// This file serves as a minimal, progressively-enhanced entry point for the
+// Login.gov Design System component integration. It serves two purposes:
+//
+// 1. Require explicit opt-in to components, to avoid behaviors from being
+//    applied to legacy (ad hoc) implementations.
+// 2. Import the minimal behavior code for components used in the application.
+//
+// The specific code (notably polyfills and configuration) follows that of the
+// default entry point for the design system, which can serve as reference.
+//
+// See: https://github.com/18F/identity-style-guide/blob/master/src/js/main.js
+// See: https://github.com/uswds/uswds/blob/develop/src/js/start.js
+
+import domready from 'domready';
+import 'uswds/src/js/polyfills';
+import 'uswds/src/js/config';
+import accordion from 'uswds/src/js/components/accordion';
+
+const COMPONENTS = {
+  '.usa-banner': accordion,
+};
+
+domready(() => {
+  Object.keys(COMPONENTS).forEach((selector) => {
+    const behavior = COMPONENTS[selector];
```

Comment:
😂  so much indirection for one behavior! Looking forward to more in the future

Reply:
> 😂 so much indirection for one behavior! Looking forward to more in the future

Yep, definitely implemented it this way under the assumption of an opt-in approach to progressively adopting USWDS native component implementations.

It's also the less risky approach, though I do have to wonder what exactly might break if we would instead just load the default script and let it apply to all markup of the page 🤔  Accordions are the likely candidate here, since we use a [custom implementation](https://github.com/18F/identity-idp/blob/master/app/javascript/app/components/accordion.js) that would likely result in some conflicting / duplicated event handling. This has its own separate problems (LG-3194).

Ideally this code could be dropped altogether, where instead we just load the `identity-style-guide` script.